### PR TITLE
Hotfix to V25 for MT-1037 Adding context menu on sunburst displays

### PR DESCRIPTION
**Pull request by bxaia**
This PR merges #519 into karoocamv25 branch. 

- Adds userlog context menu on sunburst health displays

Jira: [MT-1037](https://skaafrica.atlassian.net/browse/MT-1037)

### CODE REVIEW

**Comment by bxaia** [comment]
Just realised that the initial changes were merged in a separate PR #518  and therefore will cherrypick those and rebuild dist files

File: app/d3/d3-util-service.js [hunk 1]
```diff
@@ -94,7 +94,11 @@ angular.module('katGui.d3')
                     tooltip
                         .style("top", (y + 15 + angular.element(uiViewDiv).scrollTop()) + "px")
                         .style("left", (x + 5 + angular.element(uiViewDiv).scrollLeft()) + "px");
+<<<<<<< HEAD
 
+=======
+                    tooltip.attr("sensor", fullSensorName);
+>>>>>>> e502a832... Merge pull request #518 from ska-sa/user/xinyuwu/MT-1037/add_userlog_context_menu_on_starburst_page
```
Comment (bngcebetsha) [inline app/d3/d3-util-service.js]: conflict

File: app/health/config-health-view/config-health-view.html [hunk 2]
```diff
@@ -58,7 +58,18 @@
...
+=======
+            <receptor-health-sunburst-map class="treemap-container"
+              ng-class="{'hide-menu': !vm.sensorValue}"
+              size-storage-key="configHealthDisplaySize"
+              receptor="item"
+              ng-right-click="vm.openMenuItems($event)"
+              menu-items="vm.menuItems"
+              ng-switch-when="Sunburst">
+            </receptor-health-sunburst-map>
+>>>>>>> e502a832... Merge pull request #518 from ska-sa/user/xinyuwu/MT-1037/add_userlog_context_menu_on_starburst_page
```
Comment (bngcebetsha) [inline app/health/config-health-view/config-health-view.html]: here as well

File: app/health/config-health-view/config-health-view.js [hunk 3]
```diff
@@ -55,6 +55,64 @@
...
+          };
+
+        vm.menuItems = [
+            {
+              text:"Add user log...",
+              callback: vm.openUserLog
+            },
+        ];
+
+>>>>>>> e502a832... Merge pull request #518 from ska-sa/user/xinyuwu/MT-1037/add_userlog_context_menu_on_starburst_page
```
Comment (bngcebetsha) [inline app/health/config-health-view/config-health-view.js]: here

**Pull request closed without merging**